render-release-notes.sh

```diff
@@ -126,21 +126,22 @@ render_release_notes () {
 				(() => {
 					for (let el of document.getElementsByClassName('collapsible')) {
 						let arrow = document.createElement('div');
-						arrow.innerHTML = '⯆';
+						arrow.innerHTML = '▽';
 						arrow.style.float = 'left';
 						arrow.style.position = 'relative';
 						arrow.style.left = '-1em';
 						arrow.style.top = '+1.5em';
+						arrow.style.fontSize = 'larger';
```

Comment:
Nice! I just had a look at this locally, via `./render-release-notes.sh -p`.

Maybe we should also set [`arrow.style.cursor = 'pointer';`](https://stackoverflow.com/questions/3087975/make-the-cursor-a-hand-when-a-user-hovers-over-a-list-item)?

Reply:
> I've not looked at the pointer option. ...
Ahh I think I see what is being said by the code. Explaining to self: adding that cursor style of `pointer` will mean that when the users nous cursor floats past / hovers over the character (and maybe the whole block) the cursor style will change to be a pointer of the obvious style that would suggest 'click me to expand', or something like that.

Reply:
I amended my local copy of C:\Program Files\Git\ReleaseNotes.html #L2655 as per suggestion:

`arrow.style.cursor = 'pointer';`

Got the effect desired that when the mouse cursor hovers over the triangle character(s), then the mouse cursor changes to the 'finger pointing hand' from the previous arrow cursor. So looks like it works!